src/compas_rhino/geometry/surfaces/__init__.py

```diff
@@ -1,16 +1,13 @@
 from .surface import RhinoSurface  # noqa : F401
 from .nurbs import RhinoNurbsSurface
 
-from compas.geometry import Surface
 from compas.geometry import NurbsSurface
 from compas.plugins import plugin
 
 
 @plugin(category="factories", requires=["Rhino"])
 def new_surface(cls, *args, **kwargs):
-    surface = super(Surface, cls).__new__(cls)
-    surface.__init__(*args, **kwargs)
-    return surface
+    return object.__new__(RhinoSurface)
```

Comment:
i agree that the call to `__init__` should be removed since it is not the responsibility of the `__new__` replacement. however, i think we should use `super(...).__new__` instead of `object.__new__`. in my experience, not using `super` will not work correctly in recent CPython versions...